taskmanager/taskmanager.go

```diff
@@ -1,4 +1,161 @@
 /*
-Package taskmanager is the task/job management service layer for concurrent and asynchronous tasks
+Package taskmanager is the task/job management service layer for concurrent and asynchronous tasks with cron scheduling.
 */
 package taskmanager
+
+import (
+	"context"
+
+	"github.com/BuxOrg/bux/logging"
+	"github.com/newrelic/go-agent/v3/newrelic"
+	"github.com/robfig/cron/v3"
+	"github.com/rs/zerolog"
+	taskq "github.com/vmihailenco/taskq/v3"
+)
+
+type (
+
+	// TaskManager implements the Tasker interface
+	TaskManager struct {
+		options *options
+	}
+
+	// options holds all the configuration for the client
+	options struct {
+		cronService     *cron.Cron      // Internal cron job client
+		debug           bool            // For extra logs and additional debug information
```

Comment:
I believe we don't need `debug` flag anymore - just use `logger.Debug()`

Reply:
Done